testsuite/src/test/groovy/noe/modjk/JkConfiguratorTestIT.groovy

```diff
@@ -526,11 +526,20 @@ class JkConfiguratorTestIT extends TestAbstract {
         def sslConfFile = new File(facingServer.getConfDeploymentPath(), "ssl.conf")
         log.debug("Deleting ssl.conf: $sslConfFile.absolutePath")
         JBFile.delete(sslConfFile)
+        // we are not testing cgid here, lets remove cgi.conf (requires cgid with worker/event)
+        def cgiConfFile = new File(facingServer.getConfModulesDeploymentPath(), "01-cgi.conf")
+        log.debug("Deleting cgi.conf: $cgiConfFile.absolutePath")
+        JBFile.delete(cgiConfFile)
```

Comment:
it might make more sense to use fileVault here instead of deleting it.

Reply:
This got copied from the lines directly above them (deleting ssl.conf) so I guessed it was safe. If we are refactoring we should refactor both

Reply:
agreed, please put this on your todo list, ready for the next release